### ISSUE TITLE
Example not working

### ISSUE THREAD

**Issue by emilhe** [thedirtyfew/dash-keycloak#16]
A recent PR has (unintentionally) made the `prefix_callback_path` argument mandatory. If not provided, the following error arises,

```
<Flask 'py-dash'>
Traceback (most recent call last):
   File "/home/user/python/py-dash.py", line 17, in <module>
     FlaskKeycloak.from_kc_oidc_json(server, config_path=config_path)
   File
"/home/user/.local/lib/python3.9/site-packages/flask_keycloak/core.py",
line 197, in from_kc_oidc_json
     return FlaskKeycloak(app, keycloak_openid, redirect_uri,
logout_path=logout_path,
   File
"/home/user/.local/lib/python3.9/site-packages/flask_keycloak/core.py",
line 144, in __init__
     auth_middleware = AuthMiddleWare(app.wsgi_app, auth_handler,
redirect_uri, uri_whitelist,
   File
"/home/user/.local/lib/python3.9/site-packages/flask_keycloak/core.py",
line 77, in __init__
     self.callback_path = prefix_callback_path + "/keycloak/callback"
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

The fix to implement is to assign empty string as default value (instead of `None`).

**Comment by sg-rs** [comment]
Same Error here

**Comment by emilhe** [comment]
The issue should be solved by the following PR,

https://github.com/thedirtyfew/dash-keycloak/pull/21

which has been merged and included as part of the 0.0.25 release.
